src/server/string_family.cc

```diff
@@ -1294,12 +1319,41 @@ void StringFamily::DecrBy(CmdArgList args, const CommandContext& cmnd_cntx) {
   return IncrByGeneric(key, -val, cmnd_cntx.tx, cmnd_cntx.rb);
 }
 
+absl::FixedArray<optional<GetResp>, 8> ReorderShardResults(
```

Comment:
`absl::FixedArray<optional<GetResp>, 8>` is more than 512 bytes - and it's not moveable, meaning that now we always have to copy at least 512 bytes.
Instead, I suggest the following signature:
`void ReorderShardResults(const std::vector<MGetResponse>& mget_resp, const Transaction* t, absl::FixedArray<optional<GetResp>, 8>* dest);` 

and define res out side of the function.